services/email/index.js

```diff
@@ -0,0 +1,36 @@
+const nodemailer = require('nodemailer')
+const handlebars = require('handlebars')
+const fs = require('fs')
+const path = require('path')
+const { emailUser, emailPass } = require('../../config/local.json')
```

Comment:
Check how is it used in other places, because I think it should not be used like this, look for config function